### PR TITLE
Add lerc

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge/help-c-cpp i really don't know what is going on with cmake on windows. Any help on that front would be greatly appreciated.

**Comment by nehaljwani** [comment]
@hmaarrfk I pushed a fix for the Windows build.

**Comment by hmaarrfk** [comment]
Why did that work?

**Comment by hmaarrfk** [comment]
Had I simply forgotten to copy my CMakeLists file?

**Comment by nehaljwani** [comment]
> Had I simply forgotten to copy my CMakeLists file?

That, and the fix for the .lib location. 😉

File: recipes/lerc/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+mkdir cmake_build
+cd cmake_build
+
+cmake -LAH                            \
+    -DCMAKE_BUILD_TYPE=Release        \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX    \
+    -DCMAKE_INSTALL_LIBDIR=lib        \
+    ..
+
+make
```
Comment (scopatz) [inline recipes/lerc/build.sh:12]: `-j {CPU_COUNT}`

File: recipes/lerc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/Esri/lerc/archive/v{{ version }}.tar.gz
+  sha256: 7c48de40cd5f09319de4b39c417ff4eec4ad4b6aa5d6144f6ffa9b10d18ec94e
+  # CMakeLists.txt taken from master, and adapted for better compatibility
+  # PR made: https://github.com/Esri/lerc/pull/128
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/lerc/meta.yaml:15]: Should this have a run_exports?

Comment (hmaarrfk) [reply recipes/lerc/meta.yaml:15]: added one.

**Pull request merged**